### DISCUSSION TITLE
Github's IP addresses for Pages have been blocklisted by Spamhaus.org

### DISCUSSION THREAD

**Discussion by dylanbeattie** [community/community#127187] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been unable to access either github.io or any of my own sites hosted on GitHub Pages since yesterday.

Looks like:

1. Spamhaus.org has blacklisted the GitHub Pages IP addresses:

https://check.spamhaus.org/sbl/listings/fastly.com/

![image](https://github.com/community/community/assets/106178/bb119e05-c498-43da-9c22-3dfbf2e8a654)

2. BT UK, the backbone provider for a lot of UK internet service providers (including Plusnet and EE) is blocking traffic to IPs that are listed on Spamhaus:

https://community.bt.com/t5/BT-Fibre-broadband/DNS-Issues/td-p/2387163

Searching the forums, it looks like this has happened before:

https://github.com/orgs/community/discussions/106991
https://github.com/orgs/community/discussions/107441

and the resolution previously has been for GitHub to contact Spamhaus and get the affected sites removed.

Thanks,

Dylan

**Comment by yoannchaudet** [comment]
Are you still experiencing issues with a UK ISP?

We reached out to Spamhaus yesterday and as of this morning we should be good. Asking because we have a few IPs and Spamhaus is not exactly making it simple to have visibility into what they block.

**Comment by dylanbeattie** [comment, marked as answer]
The issue appears to be resolved as of a few hours ago; the affected IPs were still visible in Spamhaus' blocklist page at the time it was resolved so I'm guessing somebody at BT overrode the block for that set of addresses or something. Thanks for following up.
